bin/gpm

```diff
@@ -53,6 +54,17 @@ if (destOpt.value) {
     destOpt.value = Path.resolve(destOpt.value.replace("~", Ul.home()));
 }
 
+if (inputOpt.value === ".") {
+    // Load and parse the package.json file
+    var packageJsonPath = Path.join(process.cwd(), "package.json")
+      , manifest = JSON.parse(Fs.readFileSync(packageJsonPath));
+
+    // Merge all dependencies on a single Array
+    var dependencies = Object.keys(manifest.dependencies)
+      , devDependencies = Object.keys(manifest.devDependencies);
+    inputOpt.value = dependencies.concat(devDependencies);
```

Comment:
Sure, I just have some questions about the `is_dev` field, do you want me to:

- Add `devDependencies` only when `is_dev` is true.
- or
- Set `is_dev` for each package individually depending on whether it is on `devDependencies` or not.

On the first scenario, I might have to read the `package.json` inside the `Gpm`, as `Gpm` currently don't know which dependencies are devDependencies - and it is not instanced yet.

On the second scenario, `Gpm` will have to accept an `Array` of `Object`s to properly describe which dependencies are dev (this could also be used for the first scenario).

*I'll fix the `;` on the new line here too.